admin/handlers.php

```diff
@@ -188,8 +193,11 @@ function gigpress_error_checking($context) {
 				$errors['artist_name'] = __("You must enter an artist name.", "gigpress");
 			if($_POST['show_venue_id'] == 'new' && empty($_POST['venue_name']))
 				$errors['venue_name'] = __("You must enter a venue name.", "gigpress");
-			if($_POST['show_venue_id'] == 'new' && empty($_POST['venue_city']))
-				$errors['venue_city'] = __("You must enter a city.", "gigpress");
+			if ( true === $venue_city_required ) {
+				if($_POST['show_venue_id'] == 'new' && empty($_POST['venue_city'])) {
```

Comment:
Do we know at this point if the array keys exist or we should do an `isset()`?

Reply:
I think we should know since it takes the $_POST values. So either some value was sent vie those fields, or they were empty.

Reply:
I guess here we can also change `true ===` to `! empty (` or?

Reply:
I think that was done below.